src/EnergyPlus/MixedAir.cc

```diff
@@ -3341,7 +3334,7 @@ void OAControllerProps::CalcOAController(EnergyPlusData &state, int const AirLoo
         AirLoopCyclingFan = false;
     }
 
-    this->OALimitingFactor = limitFactorNone; // oa controller limiting factor
+    this->OALimitingFactor = static_cast<int>(OALimitFactor::None); // oa controller limiting factor
```

Comment:
Hmm.  Hmmmm.  Is this right?  Is limiting factor intended to be a meaningful numeric value?  If not, why is `this->OALimitingFactor` not declared as the enum type?  I have a suspicion but let's see...